src/node_report_module.cc

```diff
@@ -0,0 +1,327 @@
+#include "env.h"
+#include "node_errors.h"
+#include "node_internals.h"
+#include "node_options.h"
+#include "node_report.h"
+#include "util.h"
+
+#include "env-inl.h"
+#include "handle_wrap.h"
+#include "node_buffer.h"
+#include "stream_base-inl.h"
+#include "stream_wrap.h"
+#include "util-inl.h"
+
+#include <sstream>
+#include <atomic>
+#include <v8.h>
+
+namespace nodereport {
+using node::Environment;
+using node::FIXED_ONE_BYTE_STRING;
+using node::IsolateData;
+using node::PerIsolateOptions;
+using node::Utf8Value;
+using v8::Array;
+using v8::Boolean;
+using v8::Context;
+using v8::Function;
+using v8::FunctionCallbackInfo;
+using v8::HandleScope;
+using v8::Isolate;
+using v8::Local;
+using v8::MaybeLocal;
+using v8::Object;
+using v8::String;
+using v8::V8;
+using v8::Value;
+
+// Internal/static function declarations
+static void OnFatalError(const char* location, const char* message);
+void OnUncaughtException(const FunctionCallbackInfo<Value>& info);
+static void Initialize(Local<Object> exports,
+                       Local<Value> unused,
+                       Local<Context> context);
+
+/*******************************************************************************
+ * External JavaScript API for triggering a report
+ *
```

Comment:
Sometimes there are these extra lines in the headers, and other times not…

It’s obviously not important, but a consistent style might be nice.

(We don’t really do the `*******************************************************************************` elsewhere in core, and prefer C++-style `//` comments in general, by the way.)

Reply:
fixed; normalized with existing comment patterns, and aligned with native C++ style.